cornac/eval_methods/next_item_evaluation.py

```diff
@@ -191,8 +210,11 @@ def __init__(
             seed=seed,
             exclude_unknowns=exclude_unknowns,
             verbose=verbose,
+            mode=mode,
             **kwargs,
         )
+        assert mode in EVALUATION_MODES
```

Comment:
Please raise an exception and provide informative message with the supported eval modes. This argument is quite critical.